src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/AttachmentViewer.tsx

```diff
@@ -0,0 +1,285 @@
+  margin-top: 8px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  font-size: 16px;
+  line-height: 24px;
+  font-weight: bold;
+  text-align: center;
+  white-space: nowrap;
+`
+const CloseButton = styled(StyledButton)`
+  justify-self: end;
+`
+
+const Wrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`
+
+const PreviousButton = styled(StyledButton)`
+  position: absolute;
+  left: 0;
+  top: 50%;
+`
+
+const NextButton = styled(StyledButton)`
+  position: absolute;
+  right: 0;
+  top: 50%;
+`
+
+const Img = styled.img`
+  margin: 92px 0 20px;
+  max-width: 100%;
+  max-height: calc(100% - 112px);
+`
+
+interface Props {
+  href: string
+  attachments: Attachment[]
+  onClose: () => void
+}
+
+const AttachmentViewer: FC<Props> = ({ href, attachments, onClose }) => {
+  const wrapperRef = useRef(null)
+  const nextButtonRef = useRef(null)
+  const previousButtonRef = useRef(null)
+  const [currentHref, setCurrentHref] = useState(href)
+  const index = attachments.findIndex((item) => item.location === currentHref)
+  const previous = index > 0 ? attachments[index - 1].location : false
+  const next =
+    index < attachments.length - 1 ? attachments[index + 1].location : false
+
+  const handleKeyDown = useCallback(
+    (event) => {
+      switch (event.key) {
+        case 'ArrowLeft':
+          if (previous) {
+            setCurrentHref(previous)
+          }
+
+          break
+
+        case 'ArrowRight':
+          if (next) {
+            setCurrentHref(next)
+          }
+
+          break
+
+        default:
+          break
+      }
+    },
+    [next, previous]
+  )
+
+  /**
+   * Register and unregister listeners
+   */
+  useEffect(() => {
+    document.addEventListener('keydown', handleKeyDown)
+
+    return () => {
+      document.removeEventListener('keydown', handleKeyDown)
+    }
+  }, [handleKeyDown])
+
+  const fileName = attachments[index].location?.split('/').pop() || ''
+
+  const onClickModal = useCallback(
+    (event) => {
+      if (event.target.contains(wrapperRef.current)) {
+        onClose()
+      }
+    },
+    [onClose]
+  )
+
+  useEffect(() => {
+    if (nextButtonRef.current) (nextButtonRef.current as HTMLElement).focus()
+    else if (previousButtonRef.current)
+      (previousButtonRef.current as HTMLElement).focus()
+  }, [])
+
+  return (
+    <StyledModal
+      data-testid="modal"
+      open
+      onClose={onClose}
+      onClick={onClickModal}
+    >
+      <Header>
+        <Info>
+          {attachments[index].created_by ? (
+            <Employee>{attachments[index].created_by}</Employee>
+          ) : (
+            <Reporter>Melder</Reporter>
+          )}
+          <Date>
+            {format(
+              parseISO(attachments[index].created_at),
+              'dd-MM-yyyy HH:mm'
+            )}
+          </Date>
+        </Info>
+        <Title>{fileName}</Title>
+
+        <CloseButton
```

Comment:
can you use themeSpacing?

Reply:
I need `number`s here. `themeSpacing` returns a string with `px` suffix. Is there another way to use theme spacing?